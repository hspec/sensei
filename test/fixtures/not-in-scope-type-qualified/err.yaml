version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/not-in-scope-type-qualified/Foo.hs
  start:
    line: 2
    column: 8
  end:
    line: 2
    column: 14
severity: Error
code: 76037
message:
- |-
  Not in scope: type constructor or class `My.Map'
  Note: No module named `My' is imported.
hints: []
