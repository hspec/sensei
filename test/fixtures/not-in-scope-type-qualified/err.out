test/fixtures/not-in-scope-type-qualified/Foo.hs:2:8: error: [GHC-76037]
    Not in scope: type constructor or class `My.Map'
    NB: no module named `My' is imported.

